src/lib/date.ts

```diff
@@ -281,6 +281,13 @@ export function auctionsDetailFormattedStartDateTime(startAt, endAt, endedAt) {
   )} • ${lotsClosingMoment.format("h:mma z")}`
 }
 
+export function formattedEndDateTime(endAt) {
+  const lotEndMoment = moment.tz(endAt, "GMT")
```

Comment:
Can you double-remind: is this supposed to be hard-coding GMT for display (and the tooltip pop up for user's local time on their computer, and the tooltip is not a V1 thing)?

Reply:
[Confirmed](https://artsy.slack.com/archives/C01PBEBRB0V/p1647580848251729?thread_ts=1647370075.440409&cid=C01PBEBRB0V)